eureka/src/main/java/com/linecorp/armeria/server/eureka/EurekaUpdatingListener.java

```diff
@@ -276,14 +281,17 @@ private static String healthCheckUrl(String hostnameOrIpAddr, @Nullable String o
         if (!portWrapper.isEnabled() || !healthCheckService.isPresent()) {
             return null;
         }
+        final String baseURL = sessionProtocol.uriText() + "://" +
+                               hostnameOrIpAddr(hostnameOrIpAddr) + ':' + portWrapper.getPort();
+        if (oldHealthCheckUrlPath != null) {
+            return baseURL + oldHealthCheckUrlPath;
```

Comment:
Minor) Should we check if `oldHealthCheckUrlPath` starts with `/` when the value is first set?

Reply:
sure good idea

Reply:
PTAL